src/Microsoft.Sbom.Parsers.Spdx22SbomParser/Entities/CreationInfo.cs

```diff
@@ -28,5 +29,6 @@ public class CreationInfo
     public IEnumerable<string> Creators { get; set; }
 
     [JsonPropertyName("licenseListVersion")]
+    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
```

Comment:
Maybe flip the order of the attributes, for consistency with the others that you're adding?
 ```suggestion
    [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingNull)]
    [JsonPropertyName("licenseListVersion")]
```